components/Player/PlayerTabs.jsx

```diff
@@ -2,9 +2,9 @@ import React from 'react';
 import TabBar from '../TabBar';
 import { playerPages } from '../Header/Pages';
 
-const playerPagesMapped = (accountId) => playerPages.map((e) => Object.assign({}, e, {
-  route: `/players/${accountId}/${e.name.toLowerCase()}`,
-  label: e.name,
+const playerPagesMapped = (accountId) => playerPages.map(page => ({
+  route: `/players/${accountId}/${page.name.toLowerCase()}`,
```

Comment:
Ok. I would recommend updating your linting package. I use atom and have not had this issue before. If you are curious about atom, you can get a good list of packages to download from my github, though I would recommend changing the semvar to not be absolute. Also atom is slow as shit with large files.